doc/quickstart.rst

```diff
@@ -1,35 +1,52 @@
 Getting Started
 ===============
 
-The starting point for writing code is the
-:py:class:`QuantumProgram <qiskit.QuantumProgram>` object. The
-QuantumProgram is a collection of circuits, or scores if you are
-coming from the Quantum Experience, quantum register objects, and
-classical register objects. The QuantumProgram methods can send these
-circuits to quantum hardware or simulator backends and collect the
+The starting point for writing code is the :class:`~qiskit.QuantumCircuit`.
+A circuit (or score if you are coming from the Quantum Experience) are
+collections of :class:`~qiskit.ClassicalRegister` objects,
+:class:`~qiskit.QuantumRegister` objects and
+:mod:`gates <qiskit.extensions.standard>`. Through the
+:ref:`top-level functions <qiskit_top_level_functions>`, the circuits can be
+sent to remote quantum devices or local simulator backends and collect the
 results for further analysis.
 
 To compose and run a circuit on a simulator, which is distributed with
 this project, one can do,
 
 .. code-block:: python
 
-   from qiskit import QuantumProgram
-   qp = QuantumProgram()
+    # Import the QISKit SDK
+    from qiskit import QuantumCircuit, ClassicalRegister, QuantumRegister
+    from qiskit import available_backends, execute
 
-   qr = qp.create_quantum_register('qr', 2)
-   cr = qp.create_classical_register('cr', 2)
-   qc = qp.create_circuit('Bell', [qr], [cr])
+    # Create a Quantum Register with 2 qubits.
+    q = QuantumRegister(2)
+    # Create a Classical Register with 2 bits.
+    c = ClassicalRegister(2)
+    # Create a Quantum Circuit
+    qc = QuantumCircuit(q, c)
 
-   qc.h(qr[0])
-   qc.cx(qr[0], qr[1])
-   qc.measure(qr[0], cr[0])
-   qc.measure(qr[1], cr[1])
+    # Add a H gate on qubit 0, putting this qubit in superposition.
+    qc.h(q[0])
+    # Add a CX (CNOT) gate on control qubit 0 and target qubit 1, putting
+    # the qubits in a Bell state.
+    qc.cx(q[0], q[1])
+    # Add a Measure gate to see the state.
```

Comment:
I would say "Add a Measure gate to measure the quantum register into the classical register"

Reply:
As hinted in private, I'll avoid making this change in this PR as the idea was  to have the examples from #461 as close as possible in the three places (the example .py files, the README, and here in Sphinx) - we can revise them in a separate issue if needed.